### DISCUSSION TITLE
ScriptReference/Canvas.ForceUpdateCanvases

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#251] [Notes]
# ScriptReference/Canvas.ForceUpdateCanvases

https://docs.unity3d.com/ScriptReference/Canvas.ForceUpdateCanvases.html

**Comment by radiatoryang** [comment]
If you're looking at this page, then you probably actually want [`LayoutRebuilder.ForceLayoutRebuildImmediate()`](https://docs.unity3d.com/Packages/com.unity.ugui@latest/index.html?subfolder=/api/UnityEngine.UI.LayoutRebuilder.html?q=LayoutRebuilder#UnityEngine_UI_LayoutRebuilder_ForceRebuildLayoutImmediate_UnityEngine_RectTransform_), to force UI elements to update itself, e.g. force a deeply nested LayoutGroup to re-space its children based on new rect sizes, etc.
